lib/matterStacks.ts

```diff
@@ -55,22 +56,24 @@ export class MatterStack extends Stack {
     public static readonly matterPKITag = "matterPKITag";       // The tag that should be attached to all PAIs created in PCA.
     public static readonly matterCATypeTag = "matterCAType";    // The tag that should be attached to all CAs created in PCA and have value "paa" or "pai" only.
 
-
     public static readonly MATTER_ISSUE_PAI_ROLE_NAME =  "MatterIssuePAIRole"
     public static readonly MATTER_MANAGE_PAA_ROLE_NAME = "MatterManagePAARole"
     public static readonly MATTER_AUDITOR_ROLE_NAME = "MatterAuditorRole"
     public static readonly MATTER_AUDIT_LOGGING_BACKUP_ROLE_NAME = 'S3BackupRole'
     public static readonly MATTER_ISSUE_DAC_ROLE = "MatterIssueDACRole"
     private static readonly matterPKIRolesPath = "/MatterPKI/";
 
-    constructor(scope: Construct, id: string, prefix: string, genPaiCnt: string | undefined) {
+    constructor(scope: Construct, id: string, prefix: string, genPaiCnt: string | undefined, crlBucketName: string | undefined) {
         super(scope, id);
 
+        const root = genPaiCnt === undefined;
+
         // Note that CFN parameters are ony defined when you deploy resulting CFN template and provide values for them.
 
-        const root = genPaiCnt === undefined;
         let paaRegion: string = this.region;
         if (root) {
+            const paaCrlBucketName = prefix + (crlBucketName ?? 'matter-crl-paa-bucket');
```

Comment:
just to be consistent, please add new constants for 'matter-crl-paa-bucket' and 'matter-crl-pai-bucket' strings.

Reply:
Added